recipes/libsrtp/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ConanRecipe(ConanFile):
+    name = "libsrtp"
+    description = (
+        "This package provides an implementation of the Secure Real-time Transport"
+        "Protocol (SRTP), the Universal Security Transform (UST), and a supporting"
+        "cryptographic kernel."
+    )
+    topics = ("conan", "libsrtp", "srtp")
+    homepage = "https://github.com/cisco/libsrtp"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_openssl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_openssl": False,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def requirements(self):
+        if self.options.with_openssl:
+            self.requires("openssl/1.1.1i")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["ENABLE_OPENSSL"] = self.options.with_openssl
+        self._cmake.definitions["TEST_APPS"] = False
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "CMakeLists.txt"),
+            "install(TARGETS srtp2 DESTINATION lib)",
+            (
+                "include(GNUInstallDirs)\n"
+                "install(TARGETS srtp2\n"
+                "RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}\n"
+                "LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}\n"
+                "ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})"
+            ),
+        )
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
🤞 this is applied before the bot merges

Reply:
I do have no idea of what this does 🤔  the PR is only about v2.3, how does it relate? Do you want me to add 1.6.0 too?

Reply:
no, just to properly define pkg_config value so that pkg_config generator will create the samle file than official one for this lib, which is `libsrtp2.pc` if major version is 2.0.0
And if someone want to add version 1.x.x later, it's already properly handled

Reply:
done 👍